app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
I remember doing this change in past projects. This will definitely get the changed-file count up there in this PR.